doc/changes/changes_7.0.1.md

```diff
@@ -0,0 +1,7 @@
+# Virtual Schema Common JDBC 7.0.0, released 2020-??-??
```

Comment:
```suggestion
# Virtual Schema Common JDBC 7.0.1, released 2020-??-??
```

Reply:
Also you are breaking the interface towards the dialect, so in this case you need: 8.x.x